Checkout/Source/Tokenisation/CheckoutAPIService.swift

```diff
@@ -126,12 +149,21 @@ final public class CheckoutAPIService: CheckoutAPIProtocol {
       requestParameters,
       responseType: TokenResponse.self,
       responseErrorType: TokenisationError.ServerError.self
-    ) { [tokenDetailsFactory, logManager, logTokenResponse] tokenResponseResult, httpURLResponse in
+    ) { [riskSDK, tokenDetailsFactory, logManager, logTokenResponse] tokenResponseResult, httpURLResponse in
       logTokenResponse(tokenResponseResult, httpURLResponse)
 
       switch tokenResponseResult {
       case .response(let tokenResponse):
         let tokenDetails = tokenDetailsFactory.create(tokenResponse: tokenResponse)
+          
+          riskSDK.configure { configurationResult in
+              switch configurationResult {
+              case .failure: break
+              case .success():
+                riskSDK.publishData(cardToken: tokenDetails.token) { _ in }
+              }
+          }
+          
```

Comment:
```suggestion
    ) { [weak self] tokenResponseResult, httpURLResponse in
      guard let self else { return }
      logTokenResponse(tokenResponseResult, httpURLResponse)

      switch tokenResponseResult {
      case .response(let tokenResponse):
        let tokenDetails = tokenDetailsFactory.create(tokenResponse: tokenResponse)
          
          riskSDK.configure { configurationResult in
              switch configurationResult {
              case .failure: break
              case .success():
                self.riskSDK.publishData(cardToken: tokenDetails.token) { _ in }
              }
          }
          
```

can you please update this to be like that?